.github/actions/prefix-pr/Prefix-PR.ps1

```diff
@@ -0,0 +1,46 @@
+param(
+    [string] $Github_Repository,
+    [string] $Github_Ref,
+    [string] $Branch,
+    [string] $Github_Token
+)
+
+$jiraBaseUrl = "https://lombiq.atlassian.net/browse/"
```

Comment:
This should rather come from a parameter of the action. Supply the `DEFAULT_JIRA_BASE_URL` secret to it in our workflows, which is "https://lombiq.atlassian.net" (see how this is used by `create-jira-issues-for-community-activities`).